ExtensionExamples.md

```diff
@@ -0,0 +1,77 @@
+# Parquet extension examples
+
+To illustrate the applicability of the proposed specification we provide examples of fictional extensions to parquet and how migration can play out if/when the community decides to adopt them in the official specification.
```

Comment:
It won't be a "proposed specification" once it ends up in the format repository.

Reply:
Also, nit but "parquet" should be capitalized "Parquet".

Reply:
Fixed.